package.json

```diff
@@ -1,64 +1,65 @@
 {
-  "name": "npx",
+  "name": "app-prototype",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@types/enzyme": "^3.10.3",
-    "@types/enzyme-adapter-react-16": "^1.0.5",
-    "@types/jest": "24.0.15",
-    "@types/node": "15.3.0",
-    "@types/react": "16.8.23",
-    "@types/react-dom": "16.8.4",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
     "classnames": "^2.3.1",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.14.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-jest": "^22.6.4",
-    "eslint-plugin-prettier": "^3.1.0",
-    "husky": "^2.4.0",
-    "jest-enzyme": "^7.0.2",
-    "jest-junit": "^6.4.0",
-    "lint-staged": "^8.2.0",
-    "node-sass-chokidar": "^1.3.5",
-    "npm-run-all": "^4.1.3",
-    "prettier": "1.18.2",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-scripts": "3.0.1",
+    "eslint": "^7.23.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-jest": "^24.3.4",
+    "eslint-plugin-prettier": "^3.3.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "sass": "^1.32.8",
     "serve": "^11.0.1",
+    "typescript": "^4.1.2"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
```

Comment:
Non build-essential packages are moved into `devDependencies` to make the production `yarn install` size smaller.